### PR TITLE
Prepare for debugging of VerifyBamId

### PR DESCRIPTION
**Pull request by deflaux**
This pull request helps prepare to get the bugs out of VerifyBamId per https://github.com/googlegenomics/dataflow-java/issues/155

These test were missing ` p.run();` which means that the tests were not actually running the pipeline.  The tests are set `@Ignore` for now since they do reveal bugs.

### CODE REVIEW

File: src/test/java/com/google/cloud/genomics/dataflow/pipelines/VerifyBamIdTest.java [hunk 1]
```diff
@@ -15,6 +15,16 @@
...
+import static org.junit.Assert.assertEquals;
+
+import java.util.Map;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.beust.jcommander.internal.Lists;
```
Comment (pgrosu) [inline src/test/java/com/google/cloud/genomics/dataflow/pipelines/VerifyBamIdTest.java]: Hi Nicole,

Why do you need to import jcommander?  The code is basically the following:

```Java
(List <String>) new ArrayList<String>(Arrays.asList( (String[]) stringValue)))
```

You can even write a function for the above for the different instances used: `Lists.newArrayList("G")`, `Lists.newArrayList("T")`, and `Lists.newArrayList("C")`.

Hope it simplifies,
~p

Comment (deflaux) [inline src/test/java/com/google/cloud/genomics/dataflow/pipelines/VerifyBamIdTest.java]: Thanks @pgrosu  I tidied up the includes.

**Comment by pgrosu** [comment]
Cool!  Looks neat and tidy - thanks @deflaux :smile: 

~p

**Comment by nitag** [comment]
LGTM

**Pull request merged**